### PR TITLE
dts: msm8*16-samsung: update panel selection for fortuna

### PR DESCRIPTION
**Pull request by wonderfulShrineMaidenOfParadise**

### CODE REVIEW

**Comment by wonderfulShrineMaidenOfParadise** [comment]
```
$ ls -d SM-G530*
SM-G5306W_CHU  SM-G5309W_CTC  SM-G530FZ_DBT  SM-G530H_LYS   SM-G530M_BVO  SM-G530R4_USC  SM-G530T1_TMB  SM-G530W_XAC
SM-G5308W_TGY  SM-G530BT_ZTO  SM-G530F_XSG   SM-G530MU_TNZ  SM-G530P_SPR  SM-G530R7_ACG  SM-G530T_TMB   SM-G530Y_BRI
$ grep -rnw mdss_dsi_hx8389c_qhd_video SM-G530*/*.dts
SM-G5306W_CHU/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G5308W_TGY/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G5309W_CTC/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G530MU_TNZ/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G530Y_BRI/plat_ce-var_ce08ff01-sub_b.dts:1261:                       qcom,mdss_dsi_hx8389c_qhd_video {
```

There are only 5 models known for using `qcom,mdss_dsi_hx8389c_qhd_video`. Drop it from the other models.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
```
$ grep -rnw qcom,mdss_dsi_.* SM-G530{{6,8,9}W,MU,Y}_*/*.dtsSM-G5306W_CHU/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G5308W_TGY/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G5309W_CTC/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G530MU_TNZ/plat_ce-var_ce08ff01-sub_b.dts:1261:                      qcom,mdss_dsi_hx8389c_qhd_video {
SM-G530Y_BRI/plat_ce-var_ce08ff01-sub_b.dts:1261:                       qcom,mdss_dsi_hx8389c_qhd_video {
$ grep -rnw ss_dsi_panel.* SM-G530{{6,8,9}W,MU,Y}_*/*.dts
$
```

There is only one panel `qcom,mdss_dsi_hx8389c_qhd_video` on the models above. Drop panel selection for them.

File: lk2nd/device/dts/msm8916/msm8916-samsung.dts [hunk 1]
```diff
@@ -263,6 +260,7 @@
 		compatible = "samsung,grandprimelte", "samsung,gprimeltecan";
 		lk2nd,match-bootloader = "G530FZ*";
 
+		/* FIXME: Use msm8916-samsung-grandprimelte */
 		lk2nd,dtb-files = "msm8916-samsung-gprimeltecan";
```
Comment (TravMurav) [inline lk2nd/device/dts/msm8916/msm8916-samsung.dts:264]: This prop is a priority list so we can do this I think:

```suggestion
		lk2nd,dtb-files = "msm8916-samsung-grandprimelte",
				  "msm8916-samsung-gprimeltecan";
```

so lk2nd will first try lte then can

Comment (wonderfulShrineMaidenOfParadise) [reply lk2nd/device/dts/msm8916/msm8916-samsung.dts]: I would like to split it into another issue.

**Pull request merged**